core/package.yaml

```diff
@@ -0,0 +1,65 @@
+name:       rio-core
```

Comment:
should we name the package `mrio-core`? Its trivial to rename to `rio-core` later on upstreaming.

But we do not clash with upstream RIO meanwhile.

Reply:
Yes I agree. Lets do so.